src/portal-impl.c

```diff
@@ -691,5 +693,56 @@ find_all_portal_implementations (const char *interface)
         }
     }
 
+  if (impls->len)
```

Comment:
```suggestion
  if (impls->len > 0)
```
Just to be more clear.